wpengine/faustjs
# 1793: plugin system - take out of experimental

## Description
Make plugins official and take out of experimental mode.

## Testing
1. Review documents here: [Slack Archive of edits](https://wpengine.slack.com/archives/C02EKEWSLN6/p1708378308676539)
2. Ensure builds are successful in the PR
3. Pull and build/run the getting started project

## Documentation Changes
Documented changes are here: [Slack Archive of edits](https://wpengine.slack.com/archives/C02EKEWSLN6/p1708378308676539)

I updated the documentation to use plugins instead of experimentalPlugins and left a note for experimentalPlugins upcoming deprecation.
